### ISSUE TITLE
Allow to specify the shell

### ISSUE THREAD

**Issue by antoniomacri** [yhirose/vscode-filtertext#7]
I'm on a Windows system.

When I type "grep X" the message "Invalid command is entered" appears, so I assume the plugin uses the Windows Command prompt. Am I right?

I also have a bash on my system and I would like to use the bash instead. Can you make this configurable?

Thanks,
Antonio

**Comment by yhirose** [comment]
@antoniomacri, thanks for the request.

The extension uses the Node.js API `child_process.spawn(command[, args][, options])` to kick a command from the default shell (process.env.ComSpec on Windows). But I found the nice option in the API:

```
shell <boolean> | <string> If true, runs command inside of a shell. Uses '/bin/sh' on UNIX, and process.env.ComSpec on Windows. A different shell can be specified as a string. See Shell Requirements and Default Windows Shell. Defaults to false (no shell).
```

I'll give it a try to implement the feature.

**Comment by jpotterm** [comment]
@antoniomacri Did you try specifying the full path to `grep` (e.g. `/usr/bin/grep`)? I wouldn't expect running grep to require any shell at all; it's just an executable.

**Comment by johnnytemp** [comment]
Just found an issue bash is not in the directory of current document. Maybe let me fix first.

**Comment by yhirose** [comment]
Fixed by #10.
